loops-and-arrays.js

```diff
@@ -12,9 +14,9 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 // Na jouw script zie je de aangepaste waardes:
 // console.log(names) geeft: ["Henkje", "Pietje", "Fredje", "Joopje"]
 
-
-
-
+for ( let i= 0; i < names.length; i++ ) {
```

Comment:
Je logt hier wel de aangepaste namen in de console, maar past ze niet aan in de array.

for (let i = 0; i < names.length; i++) {
  names[i] = names[i] + 'je';
}